cas-server-webapp/src/main/resources/templates/fragments/loginsidebar.html

```diff
@@ -22,8 +22,10 @@ <h3 th:utext="#{screen.welcome.label.loginwith}" />
                         <li th:case="'cas'" th:class="${'fa fa-sign-in'}">
                             <a th:href="${entry.redirectUrl}" th:text="${entry.name}" />
                         </li>
+                        <li th:case="*" th:class="${'fa fa-sign-in'}">
```

Comment:
What does this do? If the type is unrecognized, show it anyway? and use a generic icon?